src/app/data-analytics-dashboard/oncology/oncology-reports/oncology-aggregate-report-view/oncology-aggregate-report-view.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, OnInit, Input, SimpleChanges, OnDestroy, OnChanges } from '@angular/core';
+import { SafeResourceUrl, DomSanitizer } from '@angular/platform-browser';
+import { take } from 'rxjs/operators';
```

Comment:
Please put an empty lime before this. There was an insistance from the lead dev that we adhere to the guidlines